data/compliance_profiles/CentOS/7/disa_stig.json

```diff
@@ -3,6 +3,26 @@
   "compliance_markup::compliance_map": {
     "version": "1.0.0",
     "disa_stig": {
+      "aide::aliases": {
+        "identifiers": [
+          "RHEL-07-021600",
+          "SRG-OS-000480-GPOS-00227",
+          "CCI-000366"
+        ],
+        "value": [
+          "R = p+i+l+n+u+g+s+m+c+sha1+sha256",
```

Comment:
@lnemsick-simp `compliance_markup` for the `stig` profile should have what the STIG wants. When in `enforcing` mode, this will override that parameter per the STIG

Reply:
Got it.  I was reading the check.  Should have been reading the description.